dotnet/src/SemanticKernel.Core/TemplateEngine/Blocks/CodeBlock.cs

```diff
@@ -175,7 +175,7 @@ private KernelArguments EnrichFunctionArguments(KernelArguments arguments)
         {
             object? input = ((ITextRendering)this._tokens[1]).Render(arguments);
             // Keep previous trust information when updating the input
-            arguments[KernelArguments.InputParameterName] = input;
+            arguments["input"] = input;
```

Comment:
Why do we still need this reference to  `input`?

Reply:
Nope, this was resolved when the other PR merged in, will ajust when it conflicts.